.storybook/config.js

```diff
@@ -0,0 +1,9 @@
+import { configure } from '@storybook/react';
+
+// automatically import all files ending in *.stories.js
+const req = require.context('../stories', true, /.stories.js$/);
+function loadStories() {
```

Comment:
can we convert this to an es6 arrow function?
`const loadStories = req => req.keys().forEach(filename => req(filename));`
or something like this? Idk what this logic is doing.